js/genform.js

```diff
@@ -180,3 +180,16 @@ function checkContactValidity(el) {
         el.classList.add('is-danger');
     }
 }
+
+function autoprefixEmail(el) {
+    // Add mailto: prefix, e.g. contact@example.com -> mailto:contact@example.com
+
+    // NOTE: If this regular expression is changed in future, it should always fail if
+    // the user enters some other schema (e.g. https://) so they always have the option of overriding it.
+    // Currently this is achieved by not allowing a colon to appear in the input.
+    if (/^[a-z0-9_+.-]+@[a-z0-9_+.-]+$/i.test(el.value)) {
+        el.value = 'mailto:' + el.value;
```

Comment:
Don't know which ECMASpec you target, maybe consider using string interpolation:
```suggestion
        el.value = `mailto:${el.value}`;
```

Reply:
Yeah I honestly have no idea what version of the spec we're targeting - but the current codebase is using some rather old stuff (`var` instead of `let`) and I don't want to be the one to rock the boat... I think updating everything is for another PR with more discussion

Reply:
Agreed, opened a boilerplate issue for you: https://github.com/securitytxt/securitytxt.org/issues/84